google-cloud-translate/src/main/java/com/google/cloud/translate/TranslateOptions.java

```diff
@@ -135,8 +145,45 @@ public TranslateOptions build() {
 
   private TranslateOptions(Builder builder) {
     super(TranslateFactory.class, TranslateRpcFactory.class, builder, new TranslateDefaults());
-    this.apiKey = builder.apiKey != null ? builder.apiKey : getDefaultApiKey();
-    this.targetLanguage = firstNonNull(builder.targetLanguage, Locale.ENGLISH.getLanguage());
+    // Use following order of precedence for authentication, avoiding backend conflicts (#1405):
+    // 1. explicitly set credentials
+    // 2. explicitly set API key
+    // 3. Application Default Credentials (e.g., through GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
This is [a bit more](https://developers.google.com/identity/protocols/application-default-credentials) than just `GOOGLE_APPLICATION_CREDENTIALS`.  You should also be looking at `gcloud auth application-default` info